src/components/Timeline/src/bem-mapping.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * This file overrides the default Material UI styling by mapping their style
+ * _'keys'_ to ours using the `classes` attribute.
+ *
+ * Some components also use a `className` attribute. This is defined in the
+ * component's typescript file.
+ */
+
+export const timelineClasses = {
+  root: 'denhaag-timeline',
+};
+
+export const stepClasses = {
+  root: 'denhaag-timeline__step',
+  completed: 'denhaag-timeline__step--completed',
+};
+
+export const stepIconClasses = {
+  root: 'denhaag-timeline__step-icon',
+  colorAction: 'denhaag-timeline__step-icon--active',
+  colorSecondary: 'denhaag-timeline__step-icon--completed',
+  colorPrimary: 'denhaag-timeline__step-icon--inactive',
+  colorDisabled: 'denhaag-timeline__step-icon--disabled',
+};
```

Comment:
Why are these classes defined here in `Timeline`, when they only have impact on the icon shown. This class name is very confusing to me, as in I know what it belongs to but I have zero indication of what this class does or what it changes. In my opinion, either replace these to the `Step` component and rename them to `step-icon--disabled` or make them more descriptive of their impact to the `Timeline` component.

Reply:
They are from Material,  I cannot change them unless I rewrite the component myself

Reply:
The `colorAction`, -`Secondary`, -`Primary` and -`Disable` classes are from Material, yes. But the BEM classes make absolutely zero sense the way they are defined now. It is unclear whether these classes add styling to the `Timeline` component or to the `StepIcon` component. The `stepIconClasses` const itself is already defined inside the `Timeline` component, so specifying this in the BEM-class is redundant and kills reusability. Just `step-icon--disabled` and the like would suffice here in my opinion. @Robbert @mjcarsjens what are your opinions?